indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -267,25 +280,28 @@
                                        (debug %)
                                        nil)))
                              (remove nil?))
-        humanized-fields (map #(humanizer/collection-humanizers-elastic context %) parsed-concepts)
-        suggestion-docs (map get-suggestion-docs humanized-fields)]
-    (flatten suggestion-docs)))
+        humanized-fields (map #(humanizer/collection-humanizers-elastic context %)
+                              parsed-concepts)
+        suggestion-docs (->> (map get-suggestion-docs humanized-fields)
+                             flatten
+                             (remove anti-value-suggestion?))]
+    suggestion-docs))
+
 
 (defn- reindex-suggestions-for-provider
   "Reindex autocomplete suggestion for a given provider"
   [context provider-id]
-  (info "Reindexing latest autocomplete suggestions for provider" provider-id)
+  (info "Reindexing autocomplete suggestions for provider " provider-id)
   (let [latest-collection-batches (meta-db/find-in-batches
-                                    context
-                                    :collection
-                                    REINDEX_BATCH_SIZE
-                                    {:provider-id provider-id :latest true})
+                                   context
```

Comment:
The indentation is off. The original indentation is correct. See https://wiki.earthdata.nasa.gov/display/CMR/Coding+Style+Guidelines